### PR TITLE
Fix:Removed the drag icon from the UAG stting page

### PR DESCRIPTION
**Pull request by AbhijitNage123**
- Added unset to a cursor in CSS to remove the drag icon from the UAG admin setting page.

### CODE REVIEW

File: admin/assets/admin-menu-settings.css [hunk 1]
```diff
@@ -289,3 +289,8 @@
 .uagb-reusable-block-link > span {
 	vertical-align: text-top;
 }
+
+/* cursor */
+.js .widget .widget-top, .js .postbox .hndle {
```
Comment (vrundakansara) [inline admin/assets/admin-menu-settings.css]: The selectors are very generic. Add specific selectors to the UAG setting page.
```
.uagb-container .postbox .hndle
```

File: readme.txt [hunk 2]
```diff
@@ -155,6 +155,9 @@ When you use the Ultimate Addons for Gutenberg along with the free Astra theme,
 
 == Changelog ==
 
+= Development version =
+- Fix: Removed Drag icon from UAG Admin Setting page.
```
Comment (vrundakansara) [inline readme.txt]: This is not a development version. 
Add PR to next-release branch.

**Comment by AbhijitNage123** [comment]
Done with updating changelog and CSS change.

File: admin/assets/admin-menu-settings.css [hunk 3]
```diff
@@ -289,3 +289,8 @@
 .uagb-reusable-block-link > span {
 	vertical-align: text-top;
 }
+
+/* cursor */
+.uagb-container .postbox .hndle{
```
Comment (vrundakansara) [inline admin/assets/admin-menu-settings.css]: Do not create repetitive selectors. https://github.com/brainstormforce/ultimate-addons-for-gutenberg/blob/82c0fe5b7cd7308a8617ef9ac0ebc4695e983231/admin/assets/admin-menu-settings.css#L175

**Pull request merged**